client/src/components/buttons/Button.tsx

```diff
@@ -287,11 +288,55 @@ function RoundButtonGroup({ children }: { children: React.ReactNode[] }) {
   return <ButtonGroup className="round-button-group">{children}</ButtonGroup>;
 }
 
+/*
+ * underline Link with icon
+ */
+function UnderlineArrowLink({
+  to,
+  text,
+  title,
+}: {
+  text: string;
+  to: string;
+  title: string;
```

Comment:
Change this to support any type of tooltip content:

```suggestion
  tooltip: ReactNode;
```

This works with text or react components without any effort.